### PR TITLE
Refactor task 5.4.2 to be able to use it in Check mode

### PR DESCRIPTION
**Pull request by jvgutierrez**
This PR refactors task 5.4.2 to be able to use it in check mode (dry-run).

### CODE REVIEW

File: tasks/level-1/5.4.2.yml [hunk 1]
```diff
@@ -3,36 +3,31 @@
...
     - "5.4.2"
     - scored
 
 - name: 5.4.2 - Ensure system accounts are non-login
-  debug:
-    msg: "*** ACTION REQUIRED *** {{ audit_5_4_2.stdout }}"
-  when: 
-    - audit_5_4_2.stdout_lines is defined and audit_5_4_2.stdout_lines|length > 0
-    - not fail_on_manual_remediation_actions
+  user: name={{ item }} shell=/sbin/nologin
```
Comment (anthcourtney) [inline tasks/level-1/5.4.2.yml]: This should use yaml approach to arguments. i.e.

```
  user:
    name: "{{ item }}"
    shell: "/sbin/nologin"

**Comment by anthcourtney** [comment]
Thanks very much for this contribution. My only concern is that it makes 5.4.2 an automated remediation rather than one which is flagged for manual operation (although I appreciate that the ```cis_skip_lock_users``` list will mitigate that if properly populated, and you have provided sane defaults).

On the whole, I am for this change. It may mean I have to realign some of the 6.2.x tasks to mirror this approach if appropriate.

**Comment by jvgutierrez** [comment]
Well, if you check the current version of audit_5.4.2.sh you will see that it's already an automated remediation https://github.com/anthcourtney/ansible-role-cis-amazon-linux/blob/master/files/audit_5.4.2.sh#L5-L7:

```shell
#!/bin/bash

for user in `awk -F: '($3 < 500) {print $1 }' /etc/passwd`; do
  if [ $user != "root" ]; then
    usermod -L $user
    if [ $user != "sync" ] && [ $user != "shutdown" ] && [ $user != "halt" ]; then
      usermod -s /sbin/nologin $user
    fi
  fi
done
```
I just replicated the current behaviour in a way that is compatible with Check Mode

**Comment by jvgutierrez** [comment]
@anthcourtney I just applied the requested changes and rebased my branch

**Pull request merged**